graphrag/query/__main__.py

```diff
@@ -22,69 +23,74 @@ def __str__(self):
         return self.value
 
 
+def file_exist(path):
```

Comment:
yup, top level utils :)

Reply:
Done. Moved to the top-level utils directory.